nargiz-m/nodejs-aws-shop-be
# [Task-10] Backend For Frontend

### Main taks:
### Additional (optional) tasks:
```
To test:
1. Get products list
2. Create new product
3. Get products list - result shouldn’t have new product
4. Wait more than 2 minutes
5. Get products list - result should have new product
```
### Links:
1. Product Service API URL: https://c9c9kmpx9f.execute-api.us-east-1.amazonaws.com/prod/products
2. Cart Service API URL: https://y9436f5pg8.execute-api.us-east-1.amazonaws.com/prod/api/profile/cart
4. BFF Service API URL: 
5. To call createProduct: 
6. To call Product Service and Cart Service via BFF Service URL:
